pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/PinotConnection.java

```diff
@@ -40,11 +41,20 @@
   private PinotControllerTransport _controllerTransport;
 
   PinotConnection(String controllerURL, PinotClientTransport transport, String tenant) {
-    this(controllerURL, transport, tenant, null);
+    this(controllerURL, transport, tenant, null, new Properties());
+  }
+
+  PinotConnection(String controllerURL, PinotClientTransport transport, String tenant, Properties extraProps) {
```

Comment:
Consider following the same order of the arguments as `Connection` (put `properties` as the first argument)?